scripts/gen-env-files.sh

```diff
@@ -9,7 +9,7 @@ read -p 'Username: ' varKeycloak
 echo 'Enter a username for the API database.'
 read -p 'Username: ' varApiDb
 
-passvar=$(grep -Po 'DB_PASSWORD=\K.*$' ./database/mssql/.env)
+passvar=$(grep -Po '^DB_PASSWORD=\K.*$' ./database/mssql/.env)
```

Comment:
Looks for DB_PASSWORD at the start of the line. Prevents issues where there where multiple passwords commented out

Reply:
This makes sense to me.